src/charm.py

```diff
@@ -104,8 +110,6 @@ def __init__(self, *args):
         self.framework.observe(self.on.tracing_relation_joined, self._on_tracing_relation_joined)
         self.framework.observe(self.on.tracing_relation_changed, self._on_tracing_relation_changed)
         self.framework.observe(self.on.collect_unit_status, self._on_collect_unit_status)
-        self.framework.observe(self._ingress.on.ready, self._on_ingress_ready)
-        self.framework.observe(self._ingress.on.revoked, self._on_ingress_revoked)
```

Comment:
We don't need revoked anymore?
E.g. to update the server url we advertize over tracing relations?

Reply:
It seems that we don't have a `revoked` event to observe in case of `TraefikRouteRequirer`: https://github.com/canonical/traefik-k8s-operator/blob/main/lib/charms/traefik_route_k8s/v0/traefik_route.py#L119

Reply:
Created an issue in traefik: https://github.com/canonical/traefik-k8s-operator/issues/345

IMO Grafana doesn't handle it either at the moment so this can be dealt with separately.